html-css-js/.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
GitHub Actions is not checking your linters because you put all of the linters files and folders inside this folder. Moving forward, your linters files and folders should **ALWAYS** be in your project's root folder. Please fix this and be careful of properly setting up your linters in future projects because not having GitHub Actions check for your linters is **reason for invalidation**.